packages/shared/src/lib/constants.ts

```diff
@@ -67,3 +67,18 @@ export const heimdallUrl = isDevelopment
 
 export const bookmarkLoops = 'https://r.daily.dev/bookmarkloops';
 export const migrateUserToStreaks = 'https://r.daily.dev/streaks';
+
+export const squadCategoriesPaths = {
+  discover: '/squads',
+  featured: '/squads/discover/featured',
+  general: '/squads/discover/general',
+  web: '/squads/discover/web',
+  mobile: '/squads/discover/mobile',
+  games: '/squads/discover/games',
+  devOps: '/squads/discover/DevOps',
+  cloud: '/squads/discover/cloud',
+  career: '/squads/discover/career',
+  data: '/squads/discover/data',
+  fun: '/squads/discover/fun',
+  devtools: '/squads/discover/devtools',
+};
```

Comment:
This is just temporary?
I assume they should be dynamic based on database categories?

Reply:
The only static ones here should be `discover` and `featured`. For the other ones, we should have a new query to fetch all categories. We can keep this temporary as I am already introducing it in a separate PR.

Reply:
If you keep it just add a TODO so it doesn't get forgotten please 🙏